collector/src/main.rs

```diff
@@ -1170,12 +1180,27 @@ fn main_result() -> anyhow::Result<i32> {
                     download_from_crates_io(&target_dir, &krate, &version)?
                 }
             };
+
+            eprintln!("Category: {:?}", cmd.category);
+            add_perf_config(&target_dir, cmd.category);
+
             println!("Benchmark stored at {}", target_dir.display());
             Ok(0)
         }
     }
 }
 
+fn add_perf_config(directory: &PathBuf, category: Category) {
+    let data = serde_json::json!({
+        "category": category.to_string()
+    });
+
+    let mut file = File::create(directory.join("perf-config.json"))
+        .expect("Could not create perf-config.json file");
+    serde_json::to_writer_pretty(&mut file, &data).expect("Could not write perf-config.json file");
```

Comment:
nit: generally File wants to be wrapped in a BufWriter or similar, to avoid many tiny write syscalls.

Reply:
I was wondering whether you or someone else would find this fishy :laughing: I'll wrap it.